rectools/visuals/metrics_app.py

```diff
@@ -0,0 +1,120 @@
+import typing as tp
+
+import ipywidgets as widgets
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objs as go
+from IPython.display import display
+
+from rectools import Columns
+
+DEFAULT_WIDTH = 800
+DEFAULT_HEIGHT = 600
+
+
+class MetricsApp:
+    """
+    Jupyter widgets app for metric visualization and comparison.
+    Do not create instances of this class directly. Use `MetricsApp.construct` method instead.
+    """
+
+    def __init__(
+        self,
+        df_metrics_data: pd.DataFrame,
```

Comment:
Let's call it either `metrics_data` or `metrics_df` or (even better I think) `models_metrics`. Here and in `construct`

Reply:
IMO it's kinda obvious that metrics are the model metrics.  
I agree that `data` is redundant here.  
And IMO `df` is also redundant since we have the type annotation.  
Let's keep just `metrics`

Reply:
hmm, ok, makes a bit sense  
Let's do `model_metrics`

Reply:
Done